### PR TITLE
Created Faves container, Favourite component, and a favourite list it…

### PR DESCRIPTION
**Pull request by colinmj**
…em. Pulling favourites from the database and displaying on the favourite screen

Created faves container

Logging listings from firestore

Started faves reducer

Getting faves in the faves container

Adding faves to redux store

Created Faves page

### CODE REVIEW

**Comment by colinmj** [comment]
There are a couple small things we still need to add to the faves page, once Icons are linked I will add a heart, and once we have the navigation pulled I will use firebaseAuth to get the current user. For now I have a hard coded user Id

File: js/components/FavouriteItem/FavouriteItem.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <View>
+        <Image source={{ uri: item.pictures.pic4 }} style={styles.image} />
+      </View>
+      <View style={styles.listing}>
+        <Text style={styles.title}>{item.listingTitle}</Text>
+        <View style={styles.descriptionContainer}>
+          <Text style={styles.description}>{item.description}</Text>
+        </View>
+        <Text style={styles.lastUpdated}>
+          {' '}
```
Comment (siddharthparmar7) [inline js/components/FavouriteItem/FavouriteItem.js]: Please remove {'  '} from your code.

File: js/components/FavouriteItem/FavouriteItem.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          Last Updated:{' '}
```
Comment (siddharthparmar7) [inline js/components/FavouriteItem/FavouriteItem.js]: Please remove {' '} from your code.

File: js/index.js [hunk 3]
```diff
@@ -5,15 +5,18 @@
...
+import { Provider } from 'react-redux';
+import LocationSearchContainer from './screens/LocationSearch/';
+import Favourite from './screens/Favourite';
 
 export default class App extends Component {
   render() {
     return (
       <Provider store={Store}>
-        <Login />
+        {/* <LocationSearchContainer /> */}
```
Comment (siddharthparmar7) [inline js/index.js]: Please remove this commented code.

File: js/index.js [hunk 4]
```diff
@@ -5,15 +5,18 @@
  */
 
 import React, { Component } from 'react';
-import Login from './screens/Login';
-import { Provider } from 'react-redux';
+import { Text, View } from 'react-native';
 import Store from './redux/store';
+import { Provider } from 'react-redux';
+import LocationSearchContainer from './screens/LocationSearch/';
```
Comment (siddharthparmar7) [inline js/index.js]: Please remove this import if you are not using this component.

File: js/screens/Favourite/FavouriteContainer.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+import { fetchFaves } from '../../redux/modules/faves';
+import { connect } from 'react-redux';
+
+class FavouriteScreen extends Component {
+  constructor() {
+    super();
+  }
+
+  componentDidMount() {
+    firestoreDb
```
Comment (siddharthparmar7) [inline js/screens/Favourite/FavouriteContainer.js]: your firebase implementation should be inside a helper file.

File: js/screens/Favourite/FavouriteContainer.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Favourite from './Favourite';
+import { firebaseAuth, firestoreDb } from '../../config/firebaseConfig';
+import { fetchListings } from '../../redux/modules/listings';
+import { fetchFaves } from '../../redux/modules/faves';
+import { connect } from 'react-redux';
+
+class FavouriteScreen extends Component {
```
Comment (siddharthparmar7) [inline js/screens/Favourite/FavouriteContainer.js]: Your file and your class name should be same.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes so I can approve it. Thank you!

**Review by siddharthparmar7 — COMMENTED** [review]
Please add dependencies and assign yourself to this issues.

**Review by siddharthparmar7 — APPROVED** [review]
Thank you! Colin! Please make sure to squash your commits next time.

Thank you!

**Pull request merged**